### ISSUE TITLE
jest-dom/prefer-empty wrong detection, and wrong fix.

### ISSUE THREAD

**Issue by RIP21** [testing-library/eslint-plugin-jest-dom#61]
- `eslint-plugin-jest-dom` version: 3.0.1
- `node` version: 12.16.1
- `npm` version: 6.13.4

Broken fix and detection
This is detected as error
```ts
const DONT_KNOW_TEXT = "Don't know"
expect(textarea.innerHTML).toBe(DONT_KNOW_TEXT)
```
Fixes to
```ts
expect(textarea).toBeEmpty()
```

**Comment by benmonro** [comment]
@all-contributors please add @RIP21 for bugs

**Comment by benmonro** [comment]
@RIP21 I believe all of the issues you reported should be fixed now.  Mind confirming in the latest release?

**Comment by RIP21** [comment]
@benmonro awesome. Thank you very much! You're legend :) I'll update on Monday and see how it goes :)

**Comment by atsikov** [comment]
@benmonro I'm afraid this is not fully fixed. Template strings are not properly handled.
```
expect(statusText.innerHTML).toBe(`${value}%`)
```
Still pops up as an error

**Comment by benmonro** [comment]
@atsikov thank you for reporting, I hadn't considered that use case.  will have a fix published shortly

**Comment by benmonro** [comment]
@atsikov fix released in 3.1.6 can you please update and let me know if it fixes your issue and/or any other issues I may have missed?  many thanks -Ben

**Comment by atsikov** [comment]
Wow @benmonro that was fast! 🚀 
Thanks a lot, template with params are now treated as non-empty values.

However it still shows an error in case template doesn't have params.
Sure it's not a common case, but probably worth fixing as well.
```
expect(statusText.innerHTML).toBe(`Complete`)
```

**Comment by benmonro** [comment]
ah yeah ok that actually might take a bit more work, will have to look at that tonight, as a work around you can use `'` or `"` for now.

**Comment by benmonro** [comment]
@atsikov sorry for the delay, 3.1.7 should address your issue, but please do let me know if you find another!
